<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewWindow</class>
 <widget class="QMainWindow" name="ViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tblItems">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle_key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAuthor_key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatePublished_key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Published:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPublisher_key">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Publisher:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShortDesc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Short Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblISBN_key">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSynopsis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Synopsis:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtSynopsis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShortDesc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblISBN_val">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPublisher_val">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatePublished_val">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle_val">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAuthor_val">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCategory_key">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCategory_val">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAdd_item"/>
    <addaction name="actionDelete_item"/>
    <addaction name="actionUpdate_item"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionLog_in"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAdd_item">
   <property name="text">
    <string>Add item...</string>
   </property>
  </action>
  <action name="actionDelete_item">
   <property name="text">
    <string>Delete item...</string>
   </property>
  </action>
  <action name="actionUpdate_item">
   <property name="text">
    <string>Update item...</string>
   </property>
  </action>
  <action name="actionLog_in">
   <property name="text">
    <string>Log-in</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search items</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
